I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/day5dep.rpy", line 44, in script
    f "Anyways, I'm hungry. Let's get burgers!"
ScriptError: could not find label 'day5dep.d5dgoodretro'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/day5dep.rpy", line 44, in script
    f "Anyways, I'm hungry. Let's get burgers!"
  File "C:\Games\ren_py\renpy-8.2.0-sdk\renpy\ast.py", line 1698, in execute
    rv = renpy.game.script.lookup(target)
  File "C:\Games\ren_py\renpy-8.2.0-sdk\renpy\script.py", line 1103, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'day5dep.d5dgoodretro'.

Windows-10-10.0.22621 AMD64
Ren'Py 8.2.1.24030407
test1 1.0
Fri Apr  5 18:44:10 2024
